### ISSUE TITLE
[Util Change]: Server Info bugs

### ISSUE THREAD

**Issue by Dawsson** [flytegg/mc-utils#210]
### Util Name

Server Info

### Changes

Fix the font, fix the undefined

![image](https://github.com/flytegg/mc-utils/assets/88049488/1e9d0cc8-33f0-4f31-aa12-9963b27f8bd9)
![image](https://github.com/flytegg/mc-utils/assets/88049488/0c5caf58-a27d-42a6-bfb0-2de408bd7946)

### Extra Info

No errors in console

**Comment by ShadowThijs** [comment]
There seems to be a conflict between the _latin letter small capital_ unicode character and the font being applied. The minecraft monospace font doesn't seem to have a "F" variant, when changed to the minecraft font it shows as "normal".
The _undefined_ is coming from the fact that the code does not check if **item** from the json request is an object so when item is a string, which is the case for the motd from flyte.gg 
```
motd_json": {
    "extra": [
      {
        "color": "light_purple",
        "text": "                     ꜰʟʏᴛᴇ ѕᴍᴘ"
      },
      "\n"
    ],
```
This can be changed by changing:
```
let asString = status.motd_json.extra.reduce((acc, item) => {
   if (item.color) {
      return acc + `%${item.color}%${item.text}`;
   }
   return acc + item.text;
}, '');
```
is changed into:
```
let asString = status.motd_json.extra.reduce((acc, item) => {
    if (typeof item === 'object' && item.color) {
        return acc + `%${item.color}%${item.text}`;
    }
    return acc + (typeof item === 'object' ? item.text : item);
}, '');
 ```
 
 How the font/unicode stuff needs to be changed is above my head.

**Comment by joshbker** [comment]
This was closed by #212
